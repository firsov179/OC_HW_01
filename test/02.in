Snowflakes falling
One by one,
Time to play,
And have some fun.
Build a snowman
Snowballs, too,
Come and see what you can do.