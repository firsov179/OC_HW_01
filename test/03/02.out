S: 2
n: 11
o: 11
w: 4
f: 3
l: 7
a: 10
k: 1
e: 9
s: 5
 : 17
i: 3
g: 1
O: 1
b: 2
y: 3
,: 4
T: 1
m: 4
t: 3
p: 1
A: 1
d: 4
h: 2
v: 1
u: 3
.: 2
B: 1
C: 1
c: 1
